app/javascript/components/ansible-credentials-form/index.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { useState, useMemo } from 'react';
+import PropTypes from 'prop-types';
+import MiqFormRenderer from '@@ddf';
+import createSchema from './ansible-credential-form.schema';
+
+const AnsibleCredentialsForm = () => {
```

Comment:
If you pass the attribute in the `react` call, you can access it here by destructuring:
```js
const AnsibleCredentialsForm = ({ recordId }) => {
  console.log(recordId);
```